### DISCUSSION TITLE
Potential usage of the gitopia project

### DISCUSSION THREAD

**Discussion by 31trainman** [akash-network/community#556] [General]
Has the akash team thought about moving from GitHub to the gitopia platform for its projects?

It's basically web3 GitHub from what I can tell

Here's its website https://gitopia.com

**Comment by dominikusbrian** [comment]
Lovely! would be setting a good example for the DePIN and Web3 community! 
Looking forward to hearing more about the plan and progress, maybe also help with some housecleaning work with the moving if any of relevant work might be available.
